working/bazel/module-structure.md

```diff
@@ -0,0 +1,234 @@
+The following is a reference for [`Bazel`](https://bazel.build/) (internally
+called `Blaze`) structures modules of Dart code, especially when compared to
+external structures (i.e. via `pub`).
+
+This is *not* a proposal, nor a commitment to build out further Bazel support
+externally, but rather a reference to help guide discussions around visibility
+modifiers and terminology around modular/reusable Dart code.
+
```

Comment:
Who is this document intended for?
That is, who would care about Bazel modules at all? And why should they? (So, I guess I'm asking for motivation - to make Bazel compatibility visible in language discussions - and expected audience to be spelled out).

I'm guess it is for people like me, who has no working knowledge of Bazel or Blaze, but who work with Dart pub and language packages all day long, and who might affect how well future language changes matches the Bazel model.